var/spack/repos/builtin/packages/root/package.py

```diff
@@ -310,6 +312,16 @@ class Root(CMakePackage):
     # Feature removed in 6.26.00:
     conflicts("+vmc", when="@6.26:", msg="VMC was removed in ROOT v6.26.00.")
 
+    # See https://github.com/root-project/root/issues/11128
+    conflicts("%clang@16:", when="@:6.26.07", msg="clang 16+ support was added in 6.26.08")
+
+    # See https://github.com/root-project/root/issues/11714
+    if sys.platform == "darwin" and macos_version() >= Version("13"):
+        conflicts("@:6.26.09", msg="macOS 13 support was added in 6.26.10")
```

Comment:
Ideally this would be
```python
    conflicts("os=ventura:", when="@:6.26.09", msg="macOS 13 support was added in 6.26.10")
```
but the range `:` doesn't work with OS. If you're OK with ignoring future macOS for now, you could do
```suggestion
    conflicts("os=ventura", when="@:6.26.09", msg="macOS 13 support was added in 6.26.10")
```

Reply:
I would prefer to keep semantically correct if uglier code until os has improved enough that we can use it as you suggest.